### PR TITLE
Improve some formulations in README.md

### PR DESCRIPTION
**Pull request by xeruf**
Clarifications that seemed sensible as I read through.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2220,10 +2222,10 @@ println!("{table}");
 
 ### Semver
 
-> When you need to release a breaking change—any breaking change—you do it in a major version. Period. No excuses.
+> When you need to release a breaking change — any breaking change — you do it in a major version. Period. No excuses.
 
 We still do it.
-We often do break change on minor version bump.
+We often do breaking changea on minor version bump.
```
Comment (zhiburt) [inline README.md:2228]: `changes`?

*at least I've found something :smile:

**Review by zhiburt — COMMENTED** [review]
Thanks;
The spelling is always was (and likely will) my pitfall.

I'll merge once you correct one spelling (I think it is) or let me know and I'll fix it.

Take care

**Comment by zhiburt** [comment]
And I wonder what do you think is the README/docs OK overall?

File: README.md (hunk 1 above)
Comment (Alexdelia) [reply README.md:2228]:
```suggestion
We often do breaking changes on minor version bump.
```

File: README.md [hunk 2]
```diff
@@ -234,7 +234,7 @@ let mut table = Table::new(&data);
 table.with(Style::psql());
 ```
 
-Below is a rendered list of the preconfigured styles.
+Below is a rendered list of the incl styles.
```
Comment (Alexdelia) [inline README.md:237]: not sure `incl` is clearer

especially as a non native english speaker

File: README.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ let expected = "+------+----------------+---------------+\n\
 assert_eq!(table, expected);
 ```
 
-The next example shows a builder example.
+The next example illustrates a builder method.
```
Comment (Alexdelia) [inline README.md:171]: what `method`?

there is:
```rs
let mut builder = Builder::default();
```
and
```rs
builder.push_record(words);
```

to me it's more of a `builder` example than a `builder` `method` illustration

**Comment by Alexdelia** [comment]
I really like the changes

@xeruf thanks for them :smile:

**Comment by Alexdelia** [comment]
> And I wonder what do you think is the README/docs OK overall?

imo, quite above average

there are lot of example and learning by example is one of the best way to learn (my opinion)

File: README.md (hunk 3 above)
Comment (zhiburt) [reply README.md:171]: `Builder` used here as a `pattern`.
So maybe pattern is a better alternative here?

File: README.md (hunk 2 above)
Comment (zhiburt) [reply README.md:237]: agreed

**Comment by zhiburt** [comment]
> there are lot of example

Recently I've started to think that a lot of examples could be more `real` and easier to grasp.
As many of them are using dumb data like `[1, 2, 3]` "not usefull"  text etc and not essential settings.

I wonder if making it like using real data being more useful.

I am writing it just in case you @Alexdelia  wanted to take a look at it and maybe do improve some of them.

_______________________

And don't you want to fork from this PR and open new so we could merge it?
@xeruf seems to a bit busy but I think it'd be necessary to save his commitment.

File: README.md (hunk 3 above)
Comment (Alexdelia) [reply README.md:171]: agreed

**Comment by Alexdelia** [comment]
> > there are lot of example
> 
> Recently I've started to think that a lot of examples could be more `real` and easier to grasp. As many of them are using dumb data like `[1, 2, 3]` "not usefull" text etc and not essential settings.
> 
> There are a few good examples but many are not as good as they could be that's what I am saying.
> 
> I wonder if making it like using real data being more useful. I am writing it just in case you @Alexdelia wanted to take a look at it and maybe do improve some of them.
> 
> And don't you want to fork from this PR and open new so we could merge it? @xeruf seems to a bit busy but I think it'd be necessary to save his commitment.

I'm a lead dev and got work today (unsure about how busy later this week). But I do keep that in mind for if/when time comes.

**Comment by zhiburt** [comment]
So I did merged it 3c85d08d29c49d6c884d67833021bf893d678f00
Sorry @Alexdelia I wasnot able to find your emal so Co-Author hook didn't worked for you.

But anyhow

Thanks a lot
Take care.

**Comment by Alexdelia** [comment]
> So I did merged it [3c85d08](https://github.com/zhiburt/tabled/commit/3c85d08d29c49d6c884d67833021bf893d678f00) Sorry @Alexdelia I wasnot able to find your emal so Co-Author hook didn't worked for you.

no problem, I did not contribute much code

but thanks for the effort anyway

FYI, my email: adelille@student.42.fr

**Comment by xeruf** [comment]
Thanks, indeed got caught up in other things!
Wanted to replace `preconfigured` by `included`, apparently some characters got lost there :sweat:

**Pull request closed without merging**